src/langcheck/metrics/eval_clients/_base.py

```diff
@@ -38,12 +38,15 @@ def get_float_score(
             *,
             tqdm_description: str | None = None) -> list[float | None]:
         '''The function that transforms the unstructured assessments (i.e. long
```

Comment:
Sphinx is whitespace sensitive so it was messing up the formatting of this docstring. Fixed the formatting in the docs